File: DKC/functions/dk_verbose.c
#include "dk_verbose.h"

//###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
#ifndef DKMAIN
#define DKMAIN 1
int main(int argc, char** argv) {
	
	int exit_status;
	exit_status = dk_verbose("test dk_verbose() \n");
	exit_status = dk_verbose("test dk_verbose() with number: %d\n", VERBOSE);
	exit_status = dk_verbose("test dk_verbose() with string: %s\n", "VERBOSE string");
	return exit_status;
}
#endif

File: DKPhp/functions/dk_verbose.php
<?php

################################################################################
# dk_verbose(message)
#
#    Print a verbose message to the console
#
#    @message - The message to print
#
function dk_verbose() {
	#dk_debugFunc(1);
	include_once(str_replace("\\","/",$_SERVER['USERPROFILE'])."/digitalknob/Development/DKPhp/functions/dk_log.php");
	
	if(!isset($argv)){
		$argv = func_get_args();
	}
	
    dk_log("VERBOSE", $argv[0]);
}




###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
function DKTEST() {
	#dk_infoFunc(0);

    dk_verbose("test dk_verbose message");
}


include_once(str_replace("\\","/",$_SERVER['USERPROFILE'])."/digitalknob/Development/DKPhp/functions/DK.php");
?>